pkgs/development/libraries/agda/generic/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, mkDerivation, fetchFromGitHub, standard-library }:
+
+mkDerivation rec {
+  pname = "generic";
```

Comment:
Given the repo name and folder structure, should that maybe be upper case?

Reply:
I am copying the name in the `*.agda-lib` file, which is what we have done for other libraries (e.g. `standard-library` vs `agda-stdlib`)

Reply:
Ah yes. Eternally confusing.

Reply:
It is unfortunate, both options seem bad